### PR TITLE
Fix for choice accents issue (#274)

### PR DESCRIPTION
**Pull request by akshaycj**
This change is [Reviewable](https://reviewable.io/reviews/gorgias/gorgias-chrome/282)

### CODE REVIEW

**Comment by xarg** [comment]
Please update the title of this PR. I now have to look at what issue #274 is. Please try to make the life of your reviewer as easy as possible so we can merge this stuff.

File: src/background/js/environment.js [hunk 1]
```diff
@@ -1 +1 @@
-var ENV = "production";
```
Comment (xarg) [inline src/background/js/environment.js]: again, pay attention to your diffs before you push stuff to your remote branch. This file should not be committed.

Comment (akshaycj) [reply src/background/js/environment.js]: That file is already in the .gitignore. Then how is it added to the commit?
Should i run these before i push?
```
git add -u
git reset -- src/backgrount/js/environment.js
```

**Comment by ghinda** [comment]
This could cause issues with spaces and other entities:
https://www.drupal.org/project/wysiwyg/issues/373542

We'll revisit later after we add some tests. Closing for now.

**Pull request closed without merging**